Remotion/Core/Extensions/Utilities/MemberAccessUtility.cs

```diff
@@ -78,8 +78,10 @@ public static object GetFieldOrPropertyValue (object obj, string fieldOrProperty
     public static object GetFieldOrPropertyValue (object obj, string fieldOrPropertyName, BindingFlags bindingFlags)
     {
       ArgumentUtility.CheckNotNull ("obj", obj);
+#nullable disable
       MemberInfo fieldOrProperty = GetFieldOrProperty (obj.GetType (), fieldOrPropertyName, bindingFlags, true);
       return GetFieldOrPropertyValue (obj, fieldOrProperty);
+#nullable enable

```

Comment:
@MichaelKetting  Did we agree on `!` (that's what I thought) or the disable block? I'm confused.

```suggestion
#nullable disable
      MemberInfo fieldOrProperty = GetFieldOrProperty (obj.GetType (), fieldOrPropertyName, bindingFlags, true);
      return GetFieldOrPropertyValue (obj, fieldOrProperty);
#nullable restore
```

Same question for line 111.

Reply:
The Microsoft-approach based on NFO was chosen. I just didn't want to overdo it. Since you also remarked on thsi, update with NFO, please.